_episodes_rmd/04-data-structures-part1.Rmd

```diff
@@ -241,7 +242,7 @@ a column to be the same basic data type.
 > > By keeping everything in a column the same, we allow ourselves to make simple
 > > assumptions about our data; if you can interpret one entry in the column as a
 > > number, then you can interpret *all* of them as numbers, so we don't have to
-> > check every time. This consistency, like consistently using the same separator
+> > check every time. This consistency, such as consistently using the same separator
 > > in our data files, is what people mean when they talk about *clean data*; in
```

Comment:
I might just delete this clause ("such as consistently using the same in our data files") altogether. Using a tsv and a csv in the same project doesn't mean your data are dirty.